spec/unit/webrtc/call.spec.ts

```diff
@@ -955,8 +955,42 @@ describe('Call', function() {
             await call.setScreensharingEnabled(false);
 
             expect(
-                call.getLocalFeeds().filter(f => f.purpose == SDPStreamMetadataPurpose.Screenshare).length,
-            ).toEqual(0);
+                call.getLocalFeeds().filter(f => f.purpose == SDPStreamMetadataPurpose.Screenshare),
+            ).toHaveLength(0);
         });
     });
+
+    it("falls back to replaceTrack for opponents that don't support stream metadata", async () => {
```

Comment:
Could we test `replaceTrack()` was called?

Reply:
Yeah, with a bit of re-arranging